r_markdown/emissions.Rmd

```diff
@@ -362,7 +468,7 @@ ___
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
   grav <- dplyr::mutate(grav, 
                         mass_emitted = (conc - conc_bg) * flow_hood * dur,
-                        mass_carbon = mass_emitted * 0)  # all particle carbon in ec and oc
+                        mass_carbon = mass_emitted * 0)
```

Comment:
I still think this is weird... what if we want to know the mass of carbon emitted as PM? Wouldn't it make more sense to account for it here and then filter out "grav" and other pollutants you are counting as zero when you do the calculation.